src/Camera_TEST.cc

```diff
@@ -62,6 +62,18 @@ TEST(DOMCamera, Construction)
   cam.SetFarClip(200.2);
   EXPECT_DOUBLE_EQ(200.2, cam.FarClip());
 
+  cam.SetSegmentationType("semantic");
```

Comment:
```suggestion
  EXPECT_EQUAL("semantic", cam.SegmentationType());
  cam.SetSegmentationType("panoptic");
```
The segmentation type should be `semantic` by default according to `sdf/1.9/camera.sdf`, so, can you check that the default value is correct and then test setting it to something that's not the default value?